### PR TITLE
Click handlers for footer links and brand name link

### PR DESCRIPTION
**Pull request by andrayantelo**
Resolves #59 and #60 and #57

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -435,10 +437,10 @@ <h5 id="contact-link">
                 </h4>
                 <ul class="menu-footer collapse store-info">
                     <li>
-                        <a href="#store-info">About Us</a>
+                        <a id="about-us-link" href="#">About Us</a>
                     </li>
                     <li>
-                        <a href="#store-location">Hours & Location</a>
+                        <a id="hours-location-link" href="#">Hours & Location</a>
```
Comment (johntyree) [inline docs/index.html]: You might need to html-escape this ampersand for it to work everywhere.

**Review by johntyree — CHANGES_REQUESTED** [review]

File: docs/index.html (hunk 1 above)
Comment (andrayantelo) [reply docs/index.html]: What do you mean?

Comment (andrayantelo) [reply docs/index.html]: @johntyree

Comment (johntyree) [reply docs/index.html]: https://www.w3.org/International/questions/qa-escapes#use

File: docs/ivys.js [hunk 2]
```diff
@@ -11,11 +11,74 @@ function IvyApp($grid) {
...
+        this.hideContent();
+        this.showDiv(this.$mainContent);
+    }.bind(this));
+    
 }
 
+IvyApp.prototype.hideContent = function () {
+    // Hide all content divs
+    "use strict";
+    var div;
```
Comment (johntyree) [inline docs/ivys.js:66]: Why is this here?

**Review by johntyree — CHANGES_REQUESTED** [review]

File: docs/index.html (hunk 1 above)
Comment (andrayantelo) [reply docs/index.html]: Got it.

File: docs/ivys.js (hunk 2 above)
Comment (andrayantelo) [reply docs/ivys.js:66]: Why is what here? The function? @johntyree

Comment (johntyree) [reply docs/ivys.js:66]: The div declaration, but I guess jslint just wants them all up front

File: docs/index.html [hunk 3]
```diff
@@ -440,7 +442,7 @@ <h5 id="contact-link">
                         <a id="about-us-link" href="#">About Us</a>
                     </li>
                     <li>
-                        <a id="hours-location-link" href="#">Hours & Location</a>
+                        <a id="hours-location-link" href="#">Hours &amp Location</a>
```
Comment (johntyree) [inline docs/index.html]: Missing a ';' I think.

**Review by johntyree — CHANGES_REQUESTED** [review]

File: docs/index.html (hunk 3 above)
Comment (andrayantelo) [reply docs/index.html]: Added it.

**Review by johntyree — APPROVED** [review]

**Pull request merged**